### PR TITLE
AC 2.0.1-1.dev - Fix End of Lines Static Check failure

### PR DESCRIPTION
**Pull request by ernest-kr**
**What this PR does / why we need it**:
2.0.1 1.dev - Fix End of Lines in ChangeLog
**Special notes for your reviewer**:

### CODE REVIEW

File: 2.0.1/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,6 @@
 Astronomer Certified 2.0.1-1 2020-01-21
 -----------------------------------------
 
+## Bugfixes
 
-
-
+- Bugfixes Done
```
Comment (danielhoherd) [inline 2.0.1/CHANGELOG.md:8]: This is not a useful changelog entry.

Comment (ernest-kr) [reply 2.0.1/CHANGELOG.md:8]: I need to work with Kaxil on this as how I can get the exact changelog. Currently I am trying to make sure that I can generate a 2.0.1 build.

Comment (kaxil) [reply 2.0.1/CHANGELOG.md:8]: @ernest-kr I had already linked it to you on how we do it :) in the last PR

https://github.com/astronomer/ap-airflow/pull/200#discussion_r562603325

>We don't add the entire Airflow changelog here, we only add what is new compared to Airflow 2.0.1

Example: https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md#astronomer-certified-200-1-2020-12-17

Comment (ernest-kr) [reply 2.0.1/CHANGELOG.md:8]: yes that part I did not get it. So probably need to understand it clearly. When we are upto date with 2.0.1. I am not sure what else could be added to the changelog. Thats the confusion part.

Comment (kaxil) [reply 2.0.1/CHANGELOG.md]: https://github.com/astronomer/ap-airflow/pull/202/commits/518970a470dcda57119352a3298ec54bac05b1d5 -- I just pushed a commit

File: 2.0.1/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,4 @@
 Astronomer Certified 2.0.1-1 2020-01-21
```
Comment (kaxil) [inline 2.0.1/CHANGELOG.md:3]: Also we haven't released it yet, so we should add `.dev` and not have a date.

Example: https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md#astronomer-certified-200-3dev

Comment (kaxil) [reply 2.0.1/CHANGELOG.md]: fixed in https://github.com/astronomer/ap-airflow/pull/202/commits/0015555238892b8349868d0501834ef544f806ed

**Comment by kaxil** [comment]
And please make sure to follow https://chris.beams.io/posts/git-commit/ in the PR title and always only do **Squash and merge** when merging PRs to have a single commit per PR. Especially to avoid commits like "Modified as per suggestions" etc :)

image

File: 2.0.1/CHANGELOG.md (hunk 1 above)
Comment (ernest-kr) [reply 2.0.1/CHANGELOG.md]: Thanks

**Review by kaxil — APPROVED** [review]

**Pull request merged**